conf/layer.conf

```
@@ -0,0 +1,13 @@
+# We have a conf and classes directory, add to BBPATH
+BBPATH .= ":${LAYERDIR}"
+
+# We have a packages directory, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
+            ${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "switchbsp"
+BBFILE_PATTERN_switchbsp := "^${LAYERDIR}/"
+BBFILE_PRIORITY_switchbsp = "5"
+
+LAYERDEPENDS_switchbsp = "core"
```

Your README.md says this layer depends meta-oe, but your LAYERDEPENDS does not:

```suggestion
LAYERDEPENDS_switchbsp = "core"
LAYERDEPENDS_switchbsp += "openembedded-layer"
```

---

Thanks, added. I was surprised to see other 3rd party layers not enforcing this dep.